app/src/main/java/com/github/palFinderTeam/palfinder/meetups/MeetupListAdapter.kt

```diff
@@ -1,28 +1,44 @@
 package com.github.palFinderTeam.palfinder.meetups
 
+import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Filter
 import android.widget.Filterable
+import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.github.palFinderTeam.palfinder.R
+import com.github.palFinderTeam.palfinder.utils.Location
 import com.github.palFinderTeam.palfinder.utils.PrettyDate
 import com.github.palFinderTeam.palfinder.utils.SearchedFilter
+import com.github.palFinderTeam.palfinder.utils.image.ImageInstance
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 
 
 class MeetupListAdapter(private val dataSet: List<MeetUp>, val currentDataSet: MutableList<MeetUp>,
-                        private var filter: SearchedFilter<MeetUp>, private val onItemClicked: (position: Int) -> Unit) :
+                        private var filter: SearchedFilter<MeetUp>, private var currentLocation: Location,
+                        private val onItemClicked: (position: Int) -> Unit) :
     RecyclerView.Adapter<MeetupListAdapter.ViewHolder>(), Filterable {
 
+    companion object {
+        const val PARTICIPANTS_RATIO: String = "%d / %d"
+    }
+
     class ViewHolder(view: View, private val onItemClicked: (position: Int) -> Unit) :
         RecyclerView.ViewHolder(view), View.OnClickListener {
         //TODO - add some remaining fields to display
```

Comment:
What are the field you think we could add? I thought about special requirements needed for the meetup, some restrictions and maybe some kind of category

Reply:
Icons for the list could be really nice, as someone who quickly scrolls through the meetups, they could be see immediately a beer icon or a football icon and and know what they are looking for. However you can use the tags already to filter, I think it might be visually more accessible.

Also if @atappy works on the join requirements we could display an extra icon showing that it's a meetup with conditions.